include/frg/string_stub.hpp

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+#if defined(__has_include) && __has_include(<string.h>)
```

Comment:
Do we need `defined(__has_include)`? All supported compilers already support `__has_include`.